rapid7_insightappsec/komand_rapid7_insightappsec/actions/get_scan_execution_details/action.py

```diff
@@ -25,6 +26,6 @@ def run(self, params={}):
             result = json.loads(response['resource'])
         except json.decoder.JSONDecodeError:
             self.logger.error(f'InsightAppSec response: {response}')
-            raise Exception('The response from InsightAppSec was not in JSON format. Contact support for help.'
-                            ' See log for more details')
+            raise PluginException(cause='The response from InsightAppSec was not in JSON format.', assistance='Contact support for help.'
```

Comment:
We should the JSON error preset for all these

Reply:
I set 
        response = "This is not going to work"
        OR 
         response = '{ name":"John "age":30 "car:"None" }' 

Then made a change to the exception to include TypeError and KeyError with
        except (json.decoder.JSONDecodeError, TypeError, KeyError):

INFO[0000] Running command:  docker run --rm -i rapid7/rapid7_insightappsec:2.0.0 run < tests/get_scan_execution_details.json
{"body": {"log": "Connect: Connecting...\nrapid7/Rapid7 InsightAppSec:2.0.0. Step name: get_scan_execution_details\nInsightAppSec response: { name\":\"John \"age\":30 \"car:\"None\" }\nAn error occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. See log for more details\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/rapid7_insightappsec_rapid7_plugin-2.0.0-py3.8.egg/komand_rapid7_insightappsec/actions/get_scan_execution_details/action.py\", line 28, in run\n    result = json.loads(response['resource'])\nTypeError: string indices must be integers\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-4.2.0-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 326, in handle_step\n    output = self.start_step(\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-4.2.0-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 476, in start_step\n    output = func(params)\n  File \"/usr/local/lib/python3.8/site-packages/rapid7_insightappsec_rapid7_plugin-2.0.0-py3.8.egg/komand_rapid7_insightappsec/actions/get_scan_execution_details/action.py\", line 31, in run\n    raise PluginException(cause='The response from InsightAppSec was not in JSON format.', assistance='Contact support for help.'\ninsightconnect_plugin_runtime.exceptions.PluginException: An error
occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. See log for more details\n", "status": "error", "meta": {}, "error": "An error occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. See log for more details"}, "version": "v1", "type": "action_event"}
ERRO[0004] JPP: unable to apply value to string: key not found

## Assessment
### Run

<details>

```
{
  "body": {
    "error": "An error occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. See log for
more details",
    "log": "Connect: Connecting...\nrapid7/Rapid7 InsightAppSec:2.0.0. Step name: get_scan_execution_details\nInsightAppSec response: { name\":\"John \"ag
e\":30 \"car:\"None\" }\nAn error occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. S
ee log for more details\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/rapid7_insightappsec_rapid7_plugin-2.0.0-py3.
8.egg/komand_rapid7_insightappsec/actions/get_scan_execution_details/action.py\", line 28, in run\n    result = json.loads(response['resource'])\nTypeErro
r: string indices must be integers\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-4.2.0-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 326, in handle_step\n
    output = self.start_step(\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-4.2.0-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 476, in start_step\n    output = func(params)\n  File \"/usr/local/lib/python3.8/site-packages/rapid7_insightappsec_rapid7_plugin-2.0.0-py3.8.egg/komand_rapid7_insightappsec/actions/get_scan_execution_details/action.py\", line 31, in run\n    raise PluginException(cause='The response from InsightAppSec was not in JSON format.', assistance='Contact support for help.'\ninsightconnect_plugin_runtime.exceptions.PluginException: An error occurred during plugin execution!\n\nThe response from InsightAppSec was not in JSON format. Contact support for help. See log for more details\n",
    "meta": {},
    "status": "error"
  },
  "type": "action_event",
  "version": "v1"
}

```